### DISCUSSION TITLE
Configuration more than one whatsup account on one same environment

### DISCUSSION THREAD

**Discussion by stserakhau** [Auties00/WhatsappWeb4j#217] [General]
I see that uses home folder to persist the account data. Are the any possibilities to customize path for store when configured more than one client ?

**Comment by Auties00** [comment]
The standard serializer doesn't offer that option. It could be useful so I'll add it in 3.1.3.

**Reply by stserakhau** [reply]
thanks!!!

**Comment by angleto** [comment]
I opened the PR #219 with a refactoring of the serializers that should allow to override the serializers/deserializers.
There are issues with the lombok auto-generated classes (I reproduce these issues even before my changes), using java 17:

In particular after the build, the serializers variable is still null.
```bash
        Serializers serializers = Serializers.builder()
                .serializer(new DefaultControllerSerializerProvider())
                .deserializer(new DefaultControllerDeserializerProvider())
                .build();
```

**Reply by angleto** [reply]
Hello, will you do a release anytime soon with the changes? If I import the packet I generated, I need then to add manually all the dependencies of [Whatsapp4j](https://github.com/Auties00/Whatsapp4j) to make my sbt project build. Do you know the reason why this could happen?
Thanks

**Reply by Auties00** [reply]
no idea. btw a new release is coming out tonight

**Reply by angleto** [reply]
No problem, I managed to make it working uploading a release to sonatype. Thanks.
